internal/server/deviceserver.go

```diff
@@ -39,6 +46,33 @@ func NewDeviceServer(configManager *configuration2.Manager, registrationManager
 // Send implements the "Send" method of the Worker gRPC service.
 func (s *deviceServer) Send(_ context.Context, d *pb.Data) (*pb.Response, error) {
 	go func() {
+		//check if it is an ansible playbook message
+
+		if x, found := d.GetMetadata()["ansible-playbook"]; found && x == "true" {
+			log.Debugf("Received message %s with 'ansible-playbook' metadata", d.MessageId)
+
+			// defined how to connect to hosts
+			ansiblePlaybookConnectionOptions := &options.AnsibleConnectionOptions{
```

Comment:
TBH: I would move this to Ansible package and be like ansible.GetPlaybookCMD() that returns `playbook.AnsiblePlaybookCmd` it belongs more to the ansible package ;-)